locales/templates/app.pot

```diff
@@ -0,0 +1,622 @@
+"to a free and open web."
+msgstr ""
+
+msgctxt "first_run_page2_title"
+msgid "Browse faster than ever"
+msgstr ""
+
+msgctxt "first_run_page2_text"
+msgid ""
+"Use turbo mode to block ads and trackers so you can browse at blazing "
+"speeds. If sites seem broken, just tap off the turbo mode switch in the "
+"menu."
+msgstr ""
+
+msgctxt "first_run_page3_title"
+msgid "Save data = Save money"
+msgstr ""
+
+msgctxt "first_run_page3_text"
+msgid ""
+"Blocking ads and online trackers not only speeds up browsing, it saves "
+"data. Save even more when you block images."
+msgstr ""
+
+msgctxt "first_run_page4_title"
+msgid "Screenshot everything"
+msgstr ""
+
+msgctxt "first_run_page4_text"
+msgid ""
+"Take screenshots of entire web pages. View a screenshot offline (without "
+"using data) or tap it to revisit the web page."
+msgstr ""
+
+msgctxt "firstrun_close_button"
+msgid "OK, got it!"
+msgstr ""
+
+msgctxt "firstrun_skip_button"
+msgid "Skip"
+msgstr ""
+
+msgctxt "firstrun_next_button"
+msgid "Next"
+msgstr ""
+
+#. Menu labels
+#. Zerda Need l10n
+msgctxt "label_menu_download"
+msgid "Downloads"
+msgstr ""
+
+msgctxt "label_menu_clear_cache"
+msgid "Clear cache"
+msgstr ""
+
+msgctxt "label_menu_history"
+msgid "History"
+msgstr ""
+
+msgctxt "label_menu_share"
+msgid "Share"
+msgstr ""
+
+msgctxt "label_menu_my_shots"
+msgid "My shots"
+msgstr ""
+
+msgctxt "label_menu_capture_page"
+msgid "Capture page"
+msgstr ""
+
+msgctxt "label_menu_turbo_mode"
+msgid "Turbo mode"
+msgstr ""
+
+msgctxt "label_menu_block_ads"
+msgid "Block Ads"
+msgstr ""
+
+msgctxt "label_menu_block_image"
+msgid "Block image"
+msgstr ""
+
+msgctxt "label_menu_settings"
+msgid "Settings"
+msgstr ""
+
+#. GeolocationPermissionPrompt
+#, c-format
+msgctxt "geolocation_dialog_message"
+msgid ""
+"%1$s \n"
+"wants to use your device's location."
+msgstr ""
+
+msgctxt "geolocation_dialog_allow"
+msgid "ALLOW"
+msgstr ""
+
+msgctxt "geolocation_dialog_block"
+msgid "BLOCK"
+msgstr ""
+
+#. Setting Clear browsing data dialog
+msgctxt "setting_dialog_clear_data"
+msgid "Clear Data"
+msgstr ""
+
+msgctxt "setting_dialog_cancel"
+msgid "Cancel"
+msgstr ""
+
+msgctxt "setting_dialog_browsing_history"
+msgid "Browsing history"
+msgstr ""
+
+msgctxt "setting_dialog_search_history"
+msgid "Search history"
+msgstr ""
+
+msgctxt "setting_dialog_form_history"
+msgid "Form history"
+msgstr ""
+
+msgctxt "setting_dialog_cookies"
+msgid "Cookies"
+msgstr ""
+
+msgctxt "setting_dialog_cache"
+msgid "Cache"
+msgstr ""
+
+#. Download list
+msgctxt "no_downloads"
+msgid "No downloads"
+msgstr ""
+
+msgctxt "download_started"
+msgid "Downloading…"
+msgstr ""
+
+msgctxt "download_file_not_supported"
+msgid "File type not supported"
+msgstr ""
+
+msgctxt "cannot_find_the_file"
+msgid "Cannot find the file."
+msgstr ""
+
+msgctxt "cannot_delete_the_file"
+msgid "Cannot delete the file."
+msgstr ""
+
+#. Download status
+msgctxt "pause"
+msgid "Paused"
+msgstr ""
+
+msgctxt "pending"
```

Comment:
Please add a comment that this string refers to a download (if that's the case).

Same for the following strings. Remember that localizers only see one string and comment at a time.

Reply:
Will do update the comment.
It's the file download status.